GTMAppAuth/Sources/GTMAppAuthFetcherAuthorization+Keychain.m

```diff
@@ -33,14 +40,30 @@ + (GTMAppAuthFetcherAuthorization *)authorizationFromKeychainForName:(NSString *
 }
 
 + (BOOL)removeAuthorizationFromKeychainForName:(NSString *)keychainItemName {
-  return [GTMKeychain removePasswordFromKeychainForName:keychainItemName];
+  return [GTMAppAuthFetcherAuthorization removeAuthorizationFromKeychainForName:keychainItemName
+                                                     withDataProtectionKeychain:NO];
+}
+
++ (BOOL)removeAuthorizationFromKeychainForName:(NSString *)keychainItemName
+                    withDataProtectionKeychain:(BOOL)dataProtectionKeychain {
+  return [GTMKeychain removePasswordFromKeychainForName:keychainItemName
+                             withDataProtectionKeychain:dataProtectionKeychain];
 }
 
 + (BOOL)saveAuthorization:(GTMAppAuthFetcherAuthorization *)auth
         toKeychainForName:(NSString *)keychainItemName {
+  return [GTMAppAuthFetcherAuthorization saveAuthorization:auth
+                                         toKeychainForName:keychainItemName
+                                withDataProtectionKeychain:NO];
+}
+
++ (BOOL)saveAuthorization:(GTMAppAuthFetcherAuthorization *)auth
+             toKeychainForName:(NSString *)keychainItemName
+    withDataProtectionKeychain:(BOOL)dataProtectionKeychain {
```

Comment:
Same comment about the param name not sounding BOOLish.

Reply:
Resolving as a dupe of https://github.com/google/GTMAppAuth/pull/151#discussion_r863249309